pkgs/servers/seafile-server/recent-libevhtp.patch

```diff
@@ -0,0 +1,150 @@
+From 8c3ee18d3eaf3e8a00dce677386dfae649ea309e Mon Sep 17 00:00:00 2001
```

Comment:
As this patch set unfortunately caused tests to fail and we're going with the ancient libevhtp revision instead, this file can be dropped.